### ISSUE TITLE
Toggle variable validation 

### ISSUE THREAD

**Issue by romanvog** [Unleash/unleash#928]
Is it possible to add a check if unleash variable is indeed a json, and disable the option of saving it unless it is?
For example, created a variable for a toggle and selected type to be a json, but I've inserted a string instead of a json and was able to save and pull the value.

`{'name': 'test', 'payload': {'type': 'json', 'value': 'not json'}, 'weightType': 'variable', 'enabled': True}`

Since it's not a json, I would get an error when trying to parse it as a json later on from code. Obviously it's something I can validate from code, but I think it would be safer to disable the option of saving in the first place as a safety measure

**Comment by ivarconr** [comment]
yes, it would be good if the feature toggle variant UI could perform a simple JSON validation test and fail if the payload is of `type="json"` but does not pass a JSON.parse() operations. There are a few caveats with JSON.parse(), and we should look at [this solution on stack overflow](https://stackoverflow.com/a/20392392).

**Comment by aneeshrelan** [comment]
Does this need to happen only on the UI? Adding this to `unleash-frontend`?

**Comment by ivarconr** [comment]
It's good to have it in the UI. Can do some validation serverside as well.

**Comment by divikshrivastava** [comment]
I think I can add a serverside JSON validation on this, can I work on this ? @ivarconr

**Comment by ivarconr** [comment]
Yes @divikshrivastava that's possible. A natural place of doing that is to add the validation to the joi schema (https://github.com/Unleash/unleash/blob/master/src/lib/schema/feature-schema.ts#L29) if the payload type is `json`.

**Comment by divikshrivastava** [comment]
Thanks for guidance, will raise a PR for this soon, meanwhile any doc which states how to setup local dev so that I can verify the changes @ivarconr

**Comment by ivarconr** [comment]
You need posgres basically, see

- https://github.com/Unleash/unleash/blob/master/websitev2/docs/contributing/developer-guide.md
- https://github.com/Unleash/unleash/blob/master/CONTRIBUTING.md

**Comment by romanvog** [comment]
Hey @ivarconr @divikshrivastava, any update on this one? :)
